lib/src/component/resize_sensor.dart

```diff
@@ -232,6 +301,54 @@ class ResizeSensorComponent extends UiComponent<ResizeSensorProps> with _SafeAni
       ..scrollTop = _maxSensorSize;
   }
 
+  /// Call to repair / re-initialize a [ResizeSensor] that was detached from the DOM when it mounted.
+  ///
+  /// ### How do I know if I need to call this? ###
+  ///
+  /// If you have a [ResizeSensor] that is not emitting its [ResizeSensorPropsMixin.onResize] events,
+  /// then the sensor was most likely mounted detached from the DOM.
+  /// In that situation, set the [ResizeSensorPropsMixin.onDidMountDetached] callback and use a `true`
+  /// return value to give your application knowledge that a call to this method _(at a time when you are
+  /// sure that the sensor has become attached to the DOM)_ to repair the [ResizeSensorPropsMixin.onResize] behavior.
+  ///
+  /// > __See: [ResizeSensorPropsMixin.onDidMountDetached] for more information.__
+  void forceResetDetachedSensor() => _reset();
+
+  /// Returns whether the [_expandSensorRef] / [_collapseSensorRef] nodes have a `0` value
+  /// for `offsetWidth` and/or `offsetHeight`, indicating that a call to [_reset] / [forceResetDetachedSensor]
+  /// is necessary to ensure the [ResizeSensorPropsMixin.onResize] callback is emitted as expected.
+  ///
+  /// > __See: [ResizeSensorPropsMixin.onDidMountDetached] for more information.__
+  bool _needsReset() {
+    if (_expandSensorRef.scrollLeft == 0 && _expandSensorRef.scrollTop == 0) {
+      return true;
+    } else if (_collapseSensorRef.scrollLeft == 0 || _collapseSensorRef.scrollTop == 0) {
+      return true;
+    }
+
+    return false;
+  }
+
+  /// See: [ResizeSensorPropsMixin.onDidMountDetached]
+  void _checkForDetachedMount() {
+    // Only perform this check if the consumer sets the callback.
+    if (props.onDidMountDetached == null) return;
+
+    var wasMountedDetachedFromDom = _needsReset();
```

Comment:
Can we glean this information from the DOM hierachy instead, so that we avoid accessing `scrollTop`/`scrollLeft` and causing reflows when `quickMount` is `true`?

```dart
bool _isAttachedToDocument(Node node) {
  Node current = node;
  while (current != null) {
    if (current == document) return true;
    current = node.parentNode;
  }
  return false;
}

var wasMountedDetachedFromDom = !_isAttachedToDocument(findDomNode(this));